ReactAndroid/src/androidTest/java/com/facebook/react/tests/CatalystUIManagerTestCase.java

```diff
@@ -102,7 +104,7 @@ public void run() {
 
   public void testFlexUIRendered() {
     FrameLayout rootView = createRootView();
-    jsModule.renderFlexTestApplication(rootView.getId());
+    jsModule.renderFlexTestApplication(reactTagFrom(rootView));
```

Comment:
> Should this be reactTagFor or just getReactTag?

I struggled with the naming of this one because in some places there are class methods named `getReactTag`.  I think `reactTagFor(view)` makes sense.  I'll go with that.

> It returns a tag for a view, not a tag from a view, correct?

Good point.  If the view is null, it will return View.NO_ID, so `reactTagFor` makes more sense.